include/ck_tile/ops/smoothquant/pipeline/smoothquant_pipeline_problem.hpp

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
update the year

Reply:
16a2a5b8a52ff76849f2117f4ec72f6722e79e2f